dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -140,30 +342,163 @@ export default defineComponent({
     };
   },
   name: "CreateEUTaxonomyForFinancials",
-  components: { DataPointFormElement, FailedUpload, FormKit, SuccessUpload, Card },
+  components: {
+    JumpLinksSection,
+    FailedUpload,
+    FormKit,
+    SuccessUpload,
+    UploadFormHeader,
+    Card,
+
+    UploadReports,
+    BasicInformationFields,
+
+    PrimeButton,
+    Calendar,
+    MultiSelect,
+    KPIfieldSet,
+  },
 
-  data: () => ({
-    innerClass: {
-      "formkit-inner": false,
-      "p-inputwrapper": true,
+  data() {
+    return {
+      formInputsModel: {} as CompanyAssociatedDataEuTaxonomyDataForFinancials,
+      filesToUpload: [] as ExtendedFile[],
+      uploadFiles: [] as ExtendedCompanyReport[],
+      fiscalYearEnd: "" as Date | "",
+      convertedFiscalYearEnd: "",
+      reportingPeriod: new Date(),
+      assuranceData: [
+        { label: "None", value: "None" },
+        { label: "LimitedAssurance", value: "LimitedAssurance" },
+        { label: "ReasonableAssurance", value: "ReasonableAssurance" },
+      ],
+      maxFileSize: UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+      euTaxonomyKPIsModel: euTaxonomyPseudoModelAndMappings,
+      euTaxonomyKpiNameMappings,
+      euTaxonomyKpiInfoMappings,
+      checkCustomInputs,
+      formatBytesUserFriendly,
+      route: useRoute(),
+      waitingForData: false,
+      editMode: false,
+
+      postEuTaxonomyDataForFinancialsProcessed: false,
+      messageCount: 0,
+      postEuTaxonomyDataForFinancialsResponse: null as AxiosResponse<DataMetaInformation> | null,
+      humanizeString: humanizeString,
```

Comment:
This is not used at all.

Reply:
after we change assuranceData for data from Enum it is used again